no4/style.css

```
@@ -0,0 +1,215 @@
+@import "reset.css";
+
+body {
+  font-family: "Lucida Sans", "Lucida Sans Regular", "Lucida Grande",
+    "Lucida Sans Unicode", Geneva, Verdana, sans-serif;
+}
+
+main {
+  margin: 0 auto;
+  width: 450px;
+  background-color: #ffbc41;
+  height: 100vh;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+/*헤더*/
+
+header {
+  padding-top: 3em;
+  font-weight: bold;
+  color: white;
+}
+
+h2 {
+  font-size: 2em;
+}
+
+/*캘린더*/
+
+#calender {
+  background-color: #fbe95e;
+  color: black;
+  display: grid;
+  grid-template-columns: repeat(7, 3em);
+  padding: 20px;
+  border-radius: 15px;
+  margin-top: 30px;
+  font-weight: 800;
+}
+
+#one_day {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+#day {
+  margin-bottom: 15px;
+}
+
+#icon {
+  position: relative;
+}
+
+.fa-clover {
+  font-size: 35px;
+  vertical-align: middle;
+  width: 100%;
+  color: white;
+}
+
+#icon_no {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  font-size: 10px;
+}
+
+#date {
+  margin-top: 15px;
+}
+
+#icon_selected {
+  position: relative;
+}
+
+#icon_selected i {
+  font-size: 35px;
+  vertical-align: middle;
+  width: 100%;
+  color: #c093ff;
+}
+
+#icon_no_selected {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  font-size: 10px;
+}
+
+#date_selected {
+  margin-top: 15px;
+  color: #c093ff;
+}
+
+/*투두리스트*/
+
+#list {
+  position: relative;
+  right: 90px;
+}
```

> 다 완성했는데 todo list 부분이 계속 가운데에 위치하고 있어서 옮기느라 애먹었어요.. 결국 position 을 사용해서 옮기긴 했는데 더 나은 방법이 있을거 같은데 알려주세요..😭

투두리스트 부분 너비를 지정해주면 해당 너비 내에서 배치되기 때문에 가운데에 오는 이슈를 해결할 수 있습니당!ㅎㅎ
```suggestion
#list {
  width: 100%;
}
```

---

🙊🙊🙊🙊🙊 해결사 박현지....🙆🏼‍♀️💓

---

바로 고치러 갑니당.. 최고👍🏼